src/osa/scripts/closer.py

```diff
@@ -157,6 +158,12 @@ def post_process(seq_tuple):
         list_job_id = merge_dl1_datacheck(seq_list)
         daily_datacheck(daily_longterm_cmd(list_job_id))
 
+    # Wait 5 minutes for the longterm DL1 datacheck file to be produced
+    time.sleep(300)
+
+    # Update longterm DL1 datacheck file with cherenkov_transparency script
+    cherenkov_transparency()
```

Comment:
better to add a slurm dependency between both jobs as done between runwise dl1 checks and the daily longterm check jobs